src/services/inline-context/helpers.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const LINE_BREAK = '\n';
+export const EMPTY = '';
+export const SPACE = ' ';
+
+export const ignoreSpacesAfterLineBreak = (input: string): string =>
+  input.replace(/\n\s+/g, LINE_BREAK);
+
+export const convertLineBreaksIntoSpaces = (input: string): string =>
+  input.replace(/\n/g, SPACE);
+
+export const ignoreSpacesFollowingSpace = (input: string[]): string[] => {
```

Comment:
This function would benefit from a comment showing an example input and output.

Reply:
added in [0612ff9](https://github.com/Instawork/hyperview/pull/448/commits/0612ff9c7e70e32bbfa8335b0d45f23430992bf0)